app/javascript/packages/document-capture/components/file-input.tsx

```diff
@@ -100,6 +100,12 @@ interface FileInputProps {
   onError?: (message: ReactNode) => void;
 }
 
+type AriaLabelReturnType =
+  | {
+      'aria-labelledby': string;
+    }
+  | { 'aria-label': string };
```

Comment:
Should we be consistent with multi-line object vs. single-line?

```suggestion
type AriaLabelReturnType = { 'aria-labelledby': string } | { 'aria-label': string };
```

Reply:
For whatever reason, our ESLint/prettier settings are forcing the ugly way for me. I'll have to look into it.